packages/helpers/nextjs.tsx

```diff
@@ -0,0 +1,6 @@
+import { GetServerSidePropsContext } from 'next';
+
+export function urlFromContext(context: GetServerSidePropsContext): string {
```

Comment:
FYI I didn't include this in `@ifixit/helpers/nextjs` because doing so would cause a cyclic dependency between `@ifixit/sentry` and `@ifixit/helpers`.